packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -1869,7 +1869,7 @@
 				ENABLE_BITCODE = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
```

Comment:
did you remove it last time? whats going on?

Reply:
I think the last time I touched this file was on 0.66-stable, and not main? 
It keeps on showing up so I thought might as well commit it